master/static/migrations/20231129123059_run-trial-v2-create.tx.up.sql

```diff
@@ -0,0 +1,67 @@
+-- Is this run_id a bad idea as a primary key?
+-- Would it make sense for "possible performance"
+-- reasons to use a an id that won't have gaps in the sequence.
+-- My only issue is then a field called id could be mistaked for trial id.
+CREATE TABLE trials_v2 (
```

Comment:
thoughts on having ```run_id``` be a primary key

verse having a dummy id auto incremented key? 

I know some people have opinions on having an id anyway